notebooks/00_quick_start/wide_deep_movielens.ipynb

```diff
@@ -47,24 +47,23 @@
     "import sys\n",
```

Comment:
<p>You say "the hyperparameters are&nbsp;optimized for the training set."</p><p>But for the tuning there is also a validation set.</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/686/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
good catch. more specifically, we further split the training set into training / validation (instead of split the data into three from the first place). The reason was to make sure azureml_w_d notebook uses the same set of training set as the other notebooks. I will put more contextual detail to this notebook.